website/static/js/myProjects.js

```diff
@@ -812,7 +812,9 @@ var Collections = {
                             url : collection.data.node.links.self + 'node_links/', //collection.data.node.relationships.linked_nodes.links.related.href,
                             config : xhrconfig,
                             data : dataArray[index]
-                        }).then(doNext, function(result){
+                        }).then(function(){
+                            doNext();
```

Comment:
For clarity, would be good to explicitly do `doNext(false)`. Otherwise, another dev might innocently change this back to `.then(doNext)`.